samples/ipc_call_interface/CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ cmake_minimum_required(VERSION 3.10)
 add_executable(ipc_call_interface
   ${CMAKE_CURRENT_LIST_DIR}/main.c
   ${CMAKE_CURRENT_LIST_DIR}/consumer/consumer.c
-  ${CMAKE_CURRENT_LIST_DIR}/producer/producer.c
-  ${CMAKE_CURRENT_LIST_DIR}/producer/interfaces/math_interface.c
+  ${CMAKE_CURRENT_LIST_DIR}/producer/math_1_producer_1.c
```

Comment:
```suggestion
  ${CMAKE_CURRENT_LIST_DIR}/producer/math_producer_1_1.c
```

Reply:
Or some other way to get across versioning?

Reply:
I would like to make clear the version of the interface (math_1) and the version of the implementation (producer_1). 

---
In reply to: [572478905](https://github.com/Azure/azure-ulib-c/pull/60#discussion_r572478905) [](ancestors = 572478905)